frontend/src/routes/manage/Video/Single.tsx

```diff
@@ -0,0 +1,191 @@
+            <FiArrowLeft />
+            {t("manage.nav.my-videos")}
+        </LinkWithIcon>,
+    ];
+
+    return <LinkList items={items} />;
+};
+
+const query = graphql`
+    query SingleVideoManageQuery($id: ID!) {
+        ...UserData
+        event(id: $id) {
+            title
+            description
+            opencastId
+            thumbnail
+            created
+            updated
+            duration
+            canWrite
+            series { title, ...SeriesBlockSeriesData }
+            tracks { flavor resolution }
+        }
+    }
+`;
+
+type Event = NonNullable<SingleVideoManageQueryResponse["event"]>;
+
+type Props = {
+    event: Event;
+};
+
+const BREAKPOINT = 1100;
+
+const ManageSingleVideo: React.FC<Props> = ({ event }) => {
+    const { t } = useTranslation();
+
+    return <>
+        <h1>{t("manage.my-videos.video-details")}</h1>
+        <section css={{
+            width: 1100,
+            maxWidth: "100%",
+            marginBottom: 32,
+
+            [`@media(min-width: ${BREAKPOINT}px)`]: {
+                display: "flex",
+                flexDirection: "row-reverse",
+                gap: 48,
+                "& > *:first-child": {
+
+                },
+            },
+        }}>
+            <ThumbnailDateInfo event={event} />
+            <MetadataSection event={event} />
+        </section>
+        <section>
+            <TechnicalDetails event={event} />
+        </section>
+    </>;
+};
+
+const ThumbnailDateInfo: React.FC<Props> = ({ event }) => {
+    const { t, i18n } = useTranslation();
+    const created = new Date(event.created).toLocaleString(i18n.language);
+    const updated = new Date(event.updated).toLocaleString(i18n.language);
+
+    const DateValue = useCallback(({ label, value }) => (
+        <div css={{ "&:not(:last-child)": { marginBottom: 12 } }}>
+            <div css={{ color: "var(--grey40)", lineHeight: 1 }}>{label + ":"}</div>
+            <div css={{ marginLeft: 6, marginTop: 4 }}>{value}</div>
+        </div>
+    ), []);
+
+    return (
+        <div css={{
+            flex: "0 0 auto",
+            display: "flex",
+            gap: 16,
+            [`@media(min-width: ${BREAKPOINT}px)`]: {
+                flexDirection: "column",
+                padding: 16,
+                borderLeft: "1px dashed var(--grey80)",
+            },
+        }}>
+            <Thumbnail event={event} width={16 * 12} />
+            <div css={{ fontSize: 14, margin: 4 }}>
+                {/* TODO: move those translation strings somewhere more appropriate */}
+                <DateValue label={t("video.created")} value={created} />
+                <DateValue label={t("video.updated")} value={updated} />
+            </div>
+        </div>
+    );
+};
+
+const MetadataSection: React.FC<Props> = ({ event }) => {
+    const { t } = useTranslation();
+
+    return (
+        <Form noValidate onSubmit={() => {}} css={{ margin: "0px 2px", flex: "1 0 auto" }}>
```

Comment:
What is the empty `onSubmit` handler for? Also, shouldn't one at least `preventDefault`? :thinking:

Reply:
No idea. Probably a remainder from my attempts to do what `noValidate` does, before I remembered `noValidate`. Will remove.